pontoon/pretranslation/tasks.py

```diff
@@ -126,34 +128,37 @@ def pretranslate(project: Project, paths: set[str] | None):
             )
 
             if failed_checks:
-                pretranslation = get_pretranslation(
-                    entity, locale, preserve_placeables=True
-                )
-
-            if pretranslation is not None:
-                t = Translation(
-                    entity=entity,
-                    locale=locale,
-                    string=pretranslation[0],
-                    user=pretranslation[1],
-                    approved=False,
-                    pretranslated=True,
-                    active=True,
-                )
+                try:
+                    pretranslation = get_pretranslation(
+                        entity, locale, preserve_placeables=True
+                    )
+                except ValueError as e:
+                    log.info(f"Pretranslation error: {e}")
+                    continue
```

Comment:
Previously, we weren't accounting for `get_translations` failing when it's called with `preserve_placeables=True`.